tests/active-active-rhel7-proxy/data.tf

```diff
@@ -24,7 +24,7 @@ data "aws_ami" "rhel" {
 
   filter {
     name   = "name"
-    values = ["RHEL-7.9_HVM-*-x86_64-*-Hourly2-GP2"]
+    values = ["RHEL-8.8.0_HVM-*-x86_64-*-Hourly2-GP2"]
```

Comment:
🚧 If we want to update the value, we should also reflect this change in the folder and test name.